src/EnergyPlus/MixedAir.cc

```diff
@@ -831,7 +830,11 @@ namespace MixedAir {
                 // Air-based Photovoltaic-thermal flat plate collector
             } else if (SELECT_CASE_var == PVT_AirBased) { // 'SolarCollector:FlatPlate:PhotovoltaicThermal'
                 if (Sim) {
-                    SimPVTcollectors(CompIndex, FirstHVACIteration, CalledFromOutsideAirSystem, CompName);
+                    auto thisPVT = PhotovoltaicThermalCollectors::PVTCollectorStruct::factory(CompName);
```

Comment:
If this is called every time it is simulated, we'll need to extract out the factory call and hold a reference to it so we don't have to loop through the PVs and do a string match every time.  If this is only called once then we are good to go.